src/test/kotlin/org/jetbrains/research/ml/ast/transformations/util/TransformationsTest.kt

```diff
@@ -48,6 +48,7 @@ open class TransformationsTest(testDataRoot: String) : ParametrizedBaseTest(test
         ApplicationManager.getApplication().invokeAndWait {
             transformation(psiInFile, true)
         }
+        assertTrue(psiInFile.isValid)
```

Comment:
I believe we could use `com.intellij.testFramework.PsiTestUtil.checkFileStructure` here to compare the PSI after the transformation to the PSI produced by the parser from the same text.